src/input/ButtonConfigWidget.cc

```diff
@@ -61,6 +62,12 @@ void ButtonConfigWidget::init() {
 			initComboBox(box,*ent);
 		}
 	}
+
+	WheelIgnorer *wheelIgnorer = new WheelIgnorer();
+	QList<QComboBox *> widgets = this->findChildren<QComboBox *>();
+	foreach (QComboBox* b, widgets) {
```

Comment:
Not sure if `foreach` can take an auto argument, but I assume it can: `foreach (auto b, widgets) {`.
..or just use C++: `for (auto b : widgets) {`